### PR TITLE
Conda recipies.

### PR DESCRIPTION
**Pull request by brittainhard**

### CODE REVIEW

**Comment by ahmadia** [comment]
conda_recipies -> conda_recipes

File: conda_recipies/elasticsearch/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+cp -r target/bin/* ${PREFIX}/lib/elasticsearch/bin/;
+cp -r config/* ${PREFIX}/lib/elasticsearch/config/;
+cp ${RECIPE_DIR}/elasticsearch.yml ${PREFIX}/lib/elasticsearch/config/elasticsearch.yml;
+
+rm ${PREFIX}/lib/elasticsearch/bin/*.exe;
+
+chmod +x ${PREFIX}/lib/elasticsearch/bin/elasticsearch;
+
+pushd "${PREFIX}/bin"
+ln -vs "../lib/elasticsearch/bin/elasticsearch" elasticsearch
```
Comment (ahmadia) [inline conda_recipies/elasticsearch/build.sh]: this all looks right.

File: conda_recipies/elasticsearch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+source:
+    git_rev: v1.6.0
+    git_url: https://github.com/elastic/elasticsearch.git
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - apache-maven
```
Comment (ahmadia) [inline conda_recipies/elasticsearch/meta.yaml]: Just a heads-up, it looks like the version tracked in https://github.com/conda/conda-recipes/blob/master/apache-maven/build.sh has JAVA_HOME issues and will not work on a modern Ubuntu box.

Comment (ahmadia) [inline conda_recipies/elasticsearch/meta.yaml]: If you get it working on OS X I'll start tracking the Linux installation.

Comment (ahmadia) [inline conda_recipies/elasticsearch/meta.yaml]: Unless you want to get deeper into it :)

**Comment by brittainhard** [comment]
Can't believe I mispelled recipes

**Comment by wmburke** [comment]
:) Awesome.

**Pull request merged**